examples/submit-guppy-h1-h2-via-qir.ipynb

```diff
@@ -148,15 +159,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 5,
    "id": "6271cd9e-20af-451b-92b1-289fde0e83cc",
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/run/user/1002/ipykernel_14254/2897221857.py:3: DeprecationWarning: Submission of programs to remote devices from pytket-quantinuum is deprecated, and will not be possible after October 2025. Please use qnexus ( https://docs.quantinuum.com/nexus/index.html ) instead.\n",
```

Comment:
Deprecation warning is unfortunate but I guess unavoidable for now

<img width="939" height="434" alt="Screenshot 2025-08-08 at 15 44 09" src="https://github.com/user-attachments/assets/a573fd40-81c2-4fc1-a91b-f777e5f0d426" />

Is there a plan to use qnexus in future?

Reply:
Yes, but from my understanding the QIR support in qnexus is not yet working, so still waiting for that to be published.